doc/getting_started/getting_started_cartridge.rst

```diff
@@ -183,45 +186,45 @@ replica set have the same roles enabled.
 
 Let's create a replica set containing just one instance and enable your role:
 
-#. Open the cluster management web interface at http://localhost:8081.
-#. Click **Configure**.
-#. Check the role ``Hello world!`` to enable it. Notice that the role name here
-   matches the label text that you specified in the ``role_name`` parameter in
-   the ``hello-world.lua`` file.
-#. (Optionally) Specify the replica set name, for example
-   "hello-world-replica-set".
+#.  Open the cluster management web interface at http://localhost:8081.
+#.  Next to the **router** instance, click **Configure**.
+#.  Check the role ``Hello world!`` to enable it. Notice that the role name here
+    matches the label text that you specified in the ``role_name`` parameter in
+    the ``hello-world.lua`` file.
+#.  (Optionally) Specify the replica set name, for example
+    "hello-world-replica-set".
 
-   .. image:: images/cluster_create_replica_set-border-5px.png
-      :align: center
-      :scale: 40%
+    ..  image:: images/cluster_create_replica_set-border-5px.png
+        :align: center
+        :scale: 40%
 
-#. Click **Create replica set** and see the newly-created replica set
-   in the web interface.
+#.  Click **Create replica set** and see the newly-created replica set
+    in the web interface.
 
-.. image:: images/cluster_new_replica_set-border-5px.png
-   :align: center
-   :scale: 40%
+    ..  image:: images/cluster_new_replica_set-border-5px.png
+        :align: center
+        :scale: 40%
 
 Your custom role got enabled. Find the "Hello world!" message in console,
 like this:
 
-.. image:: images/cluster_hello_world_console-border-5px.png
-   :align: center
-   :scale: 40%
+..  image:: images/cluster_hello_world_console-border-5px.png
+    :align: center
+    :scale: 40%
 
 Finally, open the HTTP endpoint of this instance at
 http://localhost:8081/hello and see the reply to your GET request.
 
-.. image:: images/cluster_hello_http-border-5px.png
-   :align: center
-   :scale: 40%
+..  image:: images/cluster_hello_http-border-5px.png
+    :align: center
+    :scale: 40%
 
 Everything is up and running! What's next?
 
-* Follow :ref:`this guide <cartridge-deployment>` to set up the rest of the
-  cluster and try some cool cluster management features.
-* Get inspired with `these examples <https://github.com/tarantool/examples/>`_
-  and implement more sophisticated business logic for your role.
-* :doc:`Pack </book/cartridge/cartridge_cli/commands/pack>` your application for easy distribution.
-  Choose what you like: a DEB or RPM package, a TGZ archive, or a Docker image.
-* Read the :doc:`Cartridge documentation </book/cartridge/index>`.
+*   Follow :ref:`this guide <cartridge-deployment>` to set up the rest of the
+    cluster and try some cool cluster management features.
+*   Get inspired with `these examples <https://github.com/tarantool/examples/>`_
```

Comment:
Maybe this link instead of examples : https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_dev/